numba/cpython/randomimpl.py

```diff
@@ -625,9 +634,9 @@ def np_uniform_impl0():
 
 
 @overload(random.uniform)
-def uniform_impl1(low):
-    if isinstance(low, (types.Float, types.Integer)):
-        return lambda low: random.uniform(low, 1.0)
+def uniform_impl1(a):
+    if isinstance(a, (types.Float, types.Integer)):
+        return lambda a: random.uniform(a, 1.0)
```

Comment:
`random.uniform` expects two arguments: `a` and `b`
https://docs.python.org/3/library/random.html#random.uniform

I guess this one is not really used/tested in Numba test suite.

Reply:
The differing number of arguments are overloaded separately. `def uniform_impl2(a, b):` is below. None of these are called by name from the test suite.

Reply:
Got it. But I think `random.uniform` takes two arguments: `a` and `b`. See:

```python
$ python
Python 3.10.12 | packaged by conda-forge | (main, Jun 23 2023, 22:41:52) [Clang 15.0.7 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import random
>>> random.uniform(2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Random.uniform() missing 1 required positional argument: 'b'
```

Reply:
Okay. I can delete it, but that will go in a separate PR. It's out of scope for this change.